multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -100,6 +121,14 @@ write_files:
       -d @"/srv/kubernetes/manifests/heapster-dc.json" \
       "http://127.0.0.1:8080/apis/extensions/v1beta1/namespaces/kube-system/deployments"
 
+      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
+      -d @"/srv/kubernetes/manifests/calico-policy-agent.json" \
+      "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
```

Comment:
Looks like `calico-policy-agent.json` is being created in `/etc/kubernetes/manifests`. Also, the manifest is a Pod, and this api endpoint is for replication controllers.